### PR TITLE
Photo storage

### PR DESCRIPTION
**Pull request by juliankohlman**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Added ability to upload and process an image via `transform.js`. 
- Updated `user`, and `photo` models with additional fields `createdAT/updatedAt` using `timestamps` field. 
- Todos: 
  - add error handling to 'transform.js', 
  - begin work on connecting `transform.js` to our Upload Pictures Page.  
- New dependencies: multer, multerS3, aws, sharp

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## Change status

* [x ] Complete, tested, ready to review and merge
* [ ] Complete, but not tested (may need new tests)
* [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

* [ x] Local server / Postman
* [ ] Test B

# Checklist:

* [ x] My code follows the style guidelines of this project
* [ x] I have performed a self-review of my own code
* [ ] My code has been reviewed by at least one peer
* [ x] I have commented my code, particularly in hard-to-understand areas
* [ ] I have made corresponding changes to the documentation
* [ x] My changes generate no new warnings
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] New and existing unit tests pass locally with my changes
* [ ] There are no merge conflicts

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -44,8 +43,6 @@ server.use(passport.initialize());
 server.use(passport.session());
 // server.use(flash());
 
-server.use('/api', router);
```
Comment (samscha) [inline server.js]: I do not believe this should route should be deleted. Please correct me if I'm wrong but I do not see an api router import. **Are you able to test `/api` to see if it returns a response?**

Also, it looks like the `server.js` on this branch isn't up to date with master. **Are you able to try to pull from master or maybe just copy/paste the code from master into this branch's `server.js`?**

Comment (juliankohlman) [reply server.js]: We can resolve the merge conflicts right? That's easier than copying and pasting from diff branches

Comment (juliankohlman) [reply server.js]: I'm looking at masters server.js and that line's not apart of the file. Just pulled it down, and there's no merge conflicts.

**Comment by samscha** [comment]
I'm not seeing the `server.use('/api', router);` on this branch's `server.js`. Are you able to add it manually? (maybe the pull didn't work properly or some sort of merge conflict?). Here's the link to this branch's `server.js`: [photo-storage branch](https://github.com/Lambda-School-Labs/pic_me/blob/photo-storage/server.js)

File: server.js (hunk 1 above)
Comment (samscha) [reply server.js]: oh okay whoops didn't even see this comment. I made another comment on the pr but I probably should have done it here.. 

I am seeing it on master here: https://github.com/Lambda-School-Labs/pic_me/blob/master/server.js

**Pull request merged**